packages/terra-toggle/tests/wdio/toggle-spec.js

```diff
@@ -0,0 +1,47 @@
+/* global browser, Terra, before */
+
+describe('Toggle', () => {
+  before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
+
+  describe('Default', () => {
+    before(() => browser.url('/#/raw/tests/toggle/default-toggle'));
+
+    Terra.should.beAccessible();
+    Terra.should.matchScreenshot('closed');
+
+    it('has aria-hidden=true when collaped', () => {
+      expect(browser.getAttribute('#toggle', 'aria-hidden')).to.equal('true');
+    });
+
+    it('has aria-hidden=false when expanded', () => {
+      browser.click('#trigger-toggle');
+      expect(browser.getAttribute('#toggle', 'aria-hidden')).to.equal('false');
+    });
+
+    Terra.should.matchScreenshot('opened');
+  });
+
+  describe('Opened', () => {
+    beforeEach(() => browser.url('/#/raw/tests/toggle/open-toggle'));
+
+    Terra.should.beAccessible();
+    Terra.should.matchScreenshot();
+  });
+
+  describe('Animated', () => {
+    beforeEach(() => browser.url('/#/raw/tests/toggle/animated-toggle'));
+
+    it('disables focusable elements when closed', () => {
```

Comment:
Instead of doing attribute and nightwatch style testing, can we use screenshots here instead?

Also, if we need to ensure that specific attributes are set on an element, I think that would be better done with jest that wdio.